### DISCUSSION TITLE
Is i3status-rust no longer packable?

### DISCUSSION THREAD

**Discussion by paoloschi** [void-linux/void-packages#47817] [Q&A]
As a user of this sw, I have noticed a series of as many as 5 unaccepted pull-requests regarding version 0.30 and later of i3status-rust.
Not understanding what this difficulty consists of, can someone kindly explain it to me?
Is it a stalemate at some point solvable or not?
Thank you.

PS: Please don't ask me to try myself making a pull-request: for sure I've not the skills :-/

**Comment by classabbyamp** [comment]
nope #47870
